src/applications/vaos/tests/mocks/setup.js

```diff
@@ -273,9 +273,10 @@ export async function setTypeOfEyeCare(store, label) {
  * @async
  * @param {ReduxStore} store The Redux store to use to render the page
  * @param {string} facilityId The facility id of the facility to be selected
+ * @param {VAFacility} facilityData The facility data to use in the mock
  * @returns {string} The url path that was routed to after clicking Continue
  */
-export async function setVAFacility(store, facilityId) {
+export async function setVAFacility(store, facilityId, facilityData) {
```

Comment:
Is "facilityId" part of the "facilityData"?

Reply:
It is, but most of the time we just need to pass the id for this helper.